lib/commands/general.js

```diff
@@ -82,6 +83,30 @@ commands.getLogTypes = function () {
   return _.keys(logTypesSupported);
 };
 
+commands.getLog = function (logType) {
+  if (!_.has(logTypesSupported, logType)) {
+      throw new Error(`Unsupported log type ${logType}.` +
+                      `supported types are ${JSON.stringify(logTypesSupported)}`);
+  }
+
+  if (logType === 'logcat') {
+    return this.adb.getLogcatLogs();
+  }
+};
+
+commands.isAppInstalled = function (appPackage) {
+  return this.adb.isAppInstalled(appPackage);
+};
+
+commands.removeApp = function (appPackage) {
+  return this.adb.uninstallApk(appPackage);
+};
+
+commands.installApp = async function (appPath) {
```

Comment:
hm. i think we do, but that logic should be extracted somewhere. will work on it